src/styles.sass

```diff
@@ -12,7 +12,7 @@ html,body
   width: 100%
   margin: 0
   padding: 0
-  overflow: hidden
+  overflow: visible
```

Comment:
Overflow is hidden by design. Since we do not have a lot of information, I want all the content to fit the page.